packages/compass-query-bar/src/components/option-editor.tsx

```diff
@@ -83,6 +83,7 @@ const insightsBadgeStyles = css({
 });
 
 type OptionEditorProps = {
+  barName: string;
```

Comment:
How do you feel about renaming this property to `optionName`? We don't refer to these as editors as bars (only the whole query bar) so I'm thinking we can make it a bit more descriptive.

Reply:
Agreed, just changed it